css/features.css

```diff
@@ -0,0 +1,45 @@
+.features--pd-btm-0 {
+  padding-bottom: 0;
+}
+
+.features__list {
+  display: flex;
+  gap: 30px;
+
+  list-style: none;
+}
+
+.features__item {
+  width: 270px;
+}
+
+.features__icon-background {
+  height: 120px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 4px;
+
+  background-color: var(--section-color);
+}
+
+.features__item-title {
+  margin: 30px 0 10px;
```

Comment:
We use a **`unidirectional margin`** approach, which means we do not use margin-top to push the lower element down. Instead, use `margin-bottom` of the element above